docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -81,6 +172,252 @@ In this section, you will configure a pipeline for shipping diagnostic logs from
    * Select `allLogs`.
    * Use the Event hub namespace and Event hub name configured in previous step in destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
 
+#### Activity Logs
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit).
```

Comment:
We should mention, if the customer is already collecting activity logs for a subscription then no need to perform this step again. Also a note that they should not tag this source with  location tag since this source will contain logs from multiple regions.